src/main/java/com/bhoruka/bloodbank/model/CampModel.java

```diff
@@ -0,0 +1,23 @@
+package com.bhoruka.bloodbank.model;
+
+import java.util.Date;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.experimental.Wither;
+
+@Wither
```

Comment:
Why is wither important here? I assumed it helped only with the final attributes?

Reply:
Which actually tips me off to the other thing I missed, this class should be annotated with Value instead of Data. We should try to keep objects immutable where ever possible. The db entity objects will need setters as that is a dynamo db constraint but the domain level objects should be immutable.